<!--
  Launch file to run multi_robot_master node. Reads a list of global goals file and publishes them 
  everytime a signal (new_global_goals) arrives from `\update_global_goals` topic.

 -->
<launch>
  <arg name="global_goals_path_file" default="$(find kobuki_multi_robot)/param/global_goals/default.txt"/>
  
  <!-- Multi master slave -->
  <node pkg="kobuki_multi_robot" type="multi_robot_master" name="multi_robot_master" output="screen" >
    <param name="global_goals_path_file" type="str" value="$(arg global_goals_path_file)" />
  </node>
</launch>
